Add p0 docs pages (#1068)
kubernetes-sigs/dra-driver-nvidia-gpu: docs/reference/helm-values.md

| `resourceApiVersion` | `""` | Override the `resource.k8s.io` API version used for DRA resources. Leave empty to auto-detect from the cluster (`v1` > `v1beta2` > `v1beta1`). |
| `logVerbosity` | `"4"` | Log verbosity for all components. `0` = errors/warnings/info only; `6` = GRPC and API request detail; `7` = health check output. |
| `allowDefaultNamespace` | `false` | Allow the chart to install into the `default` namespace. |

---

## Image

| Parameter | Default | Description |
|---|---|---|
| `image.repository` | `registry.k8s.io/dra-driver-nvidia/dra-driver-nvidia-gpu` | Container image repository. |
| `image.tag` | `""` | Image tag. Empty resolves to the chart `appVersion` with a `v` prefix. |
| `image.pullPolicy` | `IfNotPresent` | Image pull policy. |
| `imagePullSecrets` | `[]` | List of image pull secret names. |

---

## Resources

| Parameter | Default | Description |
|---|---|---|
| `resources.gpus.enabled` | `true` | Enable the GPU kubelet plugin and associated `DeviceClass` objects (`gpu.nvidia.com`, `mig.nvidia.com`, `vfio.gpu.nvidia.com`). |

> | `resources.computeDomains.enabled` | `true` | Enable the ComputeDomain controller, kubelet plugin, and associated `DeviceClass` objects. |

Reviewer: deviceclasses for CD are `compute-domain-daemon.nvidia.com` and `compute-domain-default-channel.nvidia.com`.

| `webhook.priorityClassName` | `system-cluster-critical` | Pod priority class. |
| `webhook.failurePolicy` | `Fail` | How the API server handles webhook call failures: `Fail` rejects the request, `Ignore` allows it. |
| `webhook.tls.mode` | `cert-manager` | TLS certificate source: `cert-manager` or `secret`. |
| `webhook.tls.certManager.issuerType` | `selfsigned` | cert-manager issuer type: `selfsigned`, `clusterissuer`, or `issuer`. |
| `webhook.tls.certManager.issuerName` | `""` | Issuer name. Required when `issuerType` is `clusterissuer` or `issuer`. |
| `webhook.tls.secret.name` | `""` | Name of an existing secret containing `tls.crt` and `tls.key`. Used when `tls.mode: secret`. |
| `webhook.tls.secret.caBundle` | `""` | Base64-encoded CA bundle for validating the webhook TLS certificate. Required when `tls.mode: secret`. |
| `webhook.nodeSelector` | `{}` | Node selector for webhook pods. |
| `webhook.tolerations` | `[]` | Tolerations for webhook pods. |
| `webhook.affinity` | `{}` | Affinity rules for webhook pods. |
| `webhook.podAnnotations` | `{}` | Annotations added to webhook pods. |

---

## Controller

> The `compute-domain-controller` runs as a cluster-scoped Deployment. It watches `ComputeDomain` CRs and spawns per-domain DaemonSets.

Reviewer: ```suggestion
The `compute-domain-controller` runs as controller watching for `ComputeDomain` CRs and spawns per-domain DaemonSets.
```
